contrib/texmf/template.py

```diff
@@ -0,0 +1,63 @@
+# keep in sync with texlive
+pkgname = "texmf"
+pkgver = "20240312"
+pkgrel = 0
+pkgdesc = ""  # TODO
+maintainer = "Subhaditya Nath <sn03.general@gmail.com>"
+license = "GPL-3.0-or-later"
+url = "https://tug.org/texlive"
+source = f"ftp://tug.org/historic/systems/texlive/{pkgver[:4]}/texlive-{pkgver}-texmf.tar.xz"
```

Comment:
use https://ftp.tu-chemnitz.de/pub/tug/historic/systems/texlive/2024/ or something that is not an ftp link

Reply:
using https://ftp.math.utah.edu/pub/tex/historic